websites/R/Repeat/dist/metadata.json

```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/ufIgYNI.png",
   "thumbnail": "https://i.imgur.com/7RjRna8.png",
```

Comment:
Change this thumbnail to something similar to the Bounce one.

Reply:
it was there before, untouched by me, not my problem